packages/design-system/src/components/N8nTooltip/Tooltip.vue

```diff
@@ -1,7 +1,50 @@
+<template>
+	<el-tooltip v-bind="$attrs">
+		<template v-for="(_, slotName) in $slots" #[slotName]>
+			<slot :name="slotName"/>
+			<div :key="slotName" v-if="slotName === 'content' && buttons.length" :class="$style.buttons" :style="{ justifyContent: justifyButtons }">
+				<n8n-button
```

Comment:
does not need to be fixed.. but would have expected this to just be one `dismiss` button.. because we will probably reuse this always with a dismiss button.